crates/sui-transactional-test-runner/src/args.rs

```diff
@@ -92,6 +92,14 @@ pub struct ProgrammableTransactionCommand {
     pub inputs: Vec<ParsedValue<SuiExtraValueArgs>>,
 }
 
+#[derive(Debug, clap::Parser)]
+pub struct UpgradePackageCommand {
+    #[clap(long = "upgrade-capability")]
+    pub upgrade_capability: Option<u64>,
+    #[clap(long = "gas-budget")]
+    pub gas_budget: Option<u64>,
```

Comment:
Any other args come to mind that would be useful to pass through via this command?

Reply:
I think this should be fine to get started -- you may eventually want to add a `policy` flag to test compatibility, but that can come later.